kernel/rros/clock.rs

```diff
@@ -586,13 +587,91 @@ fn set_timer_value(timer: Arc<SpinLock<RrosTimer>>, value: &Itimerspec64) -> Res
     return Ok(0);
 }
 
+pub fn rros_current_rq() -> *mut rros_rq {
+    let current = rros_current();
+    unsafe { (*(*current).locked_data().get()).rq.unwrap() }
```

Comment:
Why do we use a raw pointer here? Can we use a static reference or something safe else?

Reply:
Because other parts of the code use raw pointers `*mut rros_rq`, too. Even if here is changed, there will still be many raw pointers in other places. So I think we can make changes later when refactoring.

Reply:
I will add a **TODO:** in the code.